src/resources/postcss/views/_month.pcss

```diff
@@ -4,4 +4,38 @@
  * @package TribeEventsCalendar
  *
  */
-.tribe-events {}
\ No newline at end of file
+.tribe-events {
+
+	.tribe-events-calendar-month__event-multiday {
+		background-color: rgba(var(--color-accent-primary), .1);
```

Comment:
let's move this to the inner wrapper, and apply a white background to the outer wrapper.